src/utils/transform-filters.ts

```diff
@@ -0,0 +1,74 @@
+import { CombinedFilter, Filter, FilterCombinator } from '@yext/answers-core';
+import { SelectableFilter } from '../models/utils/selectablefilter';
+
+/**
+ * Combine a given list of Filter using OR condition into a single CombinedFilter object
+ */
+function combineFiltersWithOR(filters: Filter[]): Filter | CombinedFilter {
+  if (filters.length === 1) {
+    return filters[0];
+  }
+  return {
+    combinator: FilterCombinator.OR,
+    filters: filters
+  };
+}
+
+/**
+ * Convert a list of SelectableFilter to nested filter structure use in core
```

Comment:
another wording suggestion: 'Convert a list of SelectableFilters into the nested filter structure used by Core'